test/misc/bundle-information.js

```diff
@@ -416,4 +416,95 @@ console.log(other);Promise.all([import('./dynamic1'), import('./dynamic2')]).the
 				);
 			});
 	});
+
+	it('contains correct information about rendered/removedExports', () => {
```

Comment:
I find it quite complex and harder to read/play around with to run rollup twice in the same test. Would it make sense to turn this into two separate tests, maybe with descriptions that highlight what the difference between the two runs is? That way you cannot compare the results programmatically but I think that is only a small loss.